packages/one-app-runner/README.md

```diff
@@ -0,0 +1,257 @@
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "one-app-dev:5.0.0",
+  }
+}
+```
+
+### parrot-middleware [Optional]
+
+Path to parrot dev middleware file for One App to use for Parrot mocking. This option must be used in conjunction with the `--module` option.
+
+For more information on setting up parrot dev middleware, check out the [One App Mocking APIs recipe](https://github.com/americanexpress/one-app/blob/master/docs/recipes/Mocking-API-Calls.md).
+
+Sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --parrot-middleware ../frank-lloyd-root/dev.middleware.js --module ../frank-lloyd-root --root-module-name frank-lloyd-root --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json
+```
+
+Or in `package.json`
+
+```json
+"one-amex": {
+  "runner": {
+    "parrotMiddleware": "./dev.middleware.js",
+    "modules": ["."],
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "one-app-dev:5.0.0",
+  }
+}
+```
+
+If middleware is defined in the `package.json`, you can use the `--no-parrot-middleware` option to temporarily disable it.
+
+### dev-endpoints [Optional]
+
+Path to dev endpoints file for One App to use for the One App Dev Proxy set up. This option must be used in conjunction with the `--module` option.
+
+For more information on setting up your dev endpoints file, check out the [Configuring One App with your API URLs guide](https://github.com/americanexpress/one-app/blob/master/docs/recipes/Mocking-API-Calls.md).
+
+Sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --dev-endpoints ../frank-lloyd-root/dev.endpoints.js --module ../frank-lloyd-root --root-module-name frank-lloyd-root --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json
+```
+
+Or in `package.json`
+
+```json
+"one-amex": {
+  "runner": {
+    "devEndpoints": "./dev.endpoints.js",
+    "modules": ["."],
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "one-app-dev:5.0.0",
+  }
+}
+```
+
+### output-file [Optional]
+
+Path to a file to which One App logs should be redirected to. Using this option will send all One App logs to given file and will result in no logs from One App being sent to your terminal.
+
+Sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --output-file ../one-app.log --root-module-name frank-lloyd-root --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json  --module ../frank-lloyd-root
+```
+
+Or in `package.json`
+
+```json
+"one-amex": {
+  "runner": {
+    "outputFile": "./one-app.log",
+    "modules": ["."],
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "oneAppVersion": "5.0.0"
+  }
+}
+```
+
+### envVars [Optional]
+
+Environment variables to provide to One App instance.
```

Comment:
Can this be any type of environment variables and not one-app variables only

Reply:
Yes any env var.